### PR TITLE
Chapter 1 Game 3 Notes added

### PR DESCRIPTION
**Pull request by izzi-ink**

### CODE REVIEW

**Comment by izzi-ink** [comment]
@fimion and @colabottles have a look, the last bit in the game was fuzzy, from just before we saw tha strange sign at the keep. Please help with the last detail there.

File: docs/dnd/chapter-01-game-03.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+In addition to the map that Lux and Lokur had been given, the innkeeper, Mandol Rene, also handed them a pouch that contained a blue gem. He told them that the gem would help them on their journey to the keep. When the magic properties were spent, the gem could be exchanged for 50 gold pieces.
+
+For the first hour of the journey, things were uneventful, until Molly thought she spotted three figures moving in the grass. Three gruff and wild boars were interested in causing the party some damage, but Molly's southern twang and the bow string accompanying it, stopped them wild boars right in their tracks!
+
+Another hour into the journey, and Finn plucked at the harp strings that shone with iridescence while whistling and humming a tune. Tiny birds and critters came out of the bushes and trees, perching atop his shoulder and head.
```
Comment (fimion) [inline docs/dnd/chapter-01-game-03.md]:
```suggestion
Another hour into the journey, and Finn plucked at slightly out of tune harp strings that shone with iridescence while whistling and humming a tune. Molly stopped Finn and tuned the harp. As Finn continued to play, tiny birds and critters came out of the bushes and trees, perching atop his shoulder and head.
```

File: docs/dnd/chapter-01-game-03.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## A Wild Goose Chase after Bears 
+
+## 18 February 2023
+
+Lux introduced Lokur to the rest of the party and they filled Molly and Finn in on the strange events that had occurred in the Squirming Octopus tavern, while they had been asleep. Finn welcomed Lokur to the party and with a hug and a scoop, lifted the Sea Elf into his arms. Flustered and embarrassed, Lokur tried to hide her face and caught completely off guard had no idea who "Charles the Octopus was" that Finn asked, with little room for awkwardness left, the party set off to find Jax. 
+
+In addition to the map that Lux and Lokur had been given, the innkeeper, Mandol Rene, also handed them a pouch that contained a blue gem. He told them that the gem would help them on their journey to the keep. When the magic properties were spent, the gem could be exchanged for 50 gold pieces.
+
+For the first hour of the journey, things were uneventful, until Molly thought she spotted three figures moving in the grass. Three gruff and wild boars were interested in causing the party some damage, but Molly's southern twang and the bow string accompanying it, stopped them wild boars right in their tracks!
```
Comment (fimion) [inline docs/dnd/chapter-01-game-03.md]:
```suggestion
For the first hour of the journey, things were uneventful, until Molly thought she spotted three figures moving in the grass. Three gruff and wild boars were interested in causing the party some damage, but Molly's warning gave Finn an opportunity to ask them to leave, which they did!
```

**Review by colabottles — APPROVED** [review]

**Review by fimion — APPROVED** [review]
Approved!
Approved

**Comment by colabottles** [comment]
@izzi-ink Could it be possible to give us the permissions to merge in this repo?

**Review by colabottles — APPROVED** [review]

**Pull request merged**